pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -135,29 +199,131 @@ def __init__(
             self,
             wires,
             *,
-            sync=True,
+            sync=False,
```

Comment:
Is this argument used anywhere with the updates?

Reply:
yes, it's an input for the memory copy between host and device.